client/src/App.tsx

```diff
@@ -82,7 +82,7 @@ export default function App() {
     return (
 
         <Routes>
-            <Route path="/" element={<Home login={login} logout={logout} loginErrorMessage={errorMessage} error={error} setError={setError} user={user} />} />
+            <Route path="/" element={<Home user={user} />} />
             <Route path="/dashboard" element={<Console user={user} />} />
             <Route path="/documents/:id" element={<Document user={user} />} />
```

Comment:
<img width="153" alt="Screen Shot 2024-12-13 at 18 38 36" src="https://github.com/user-attachments/assets/c41d4294-11d4-4b48-b41d-cc511bcba3fd" />

there is a white background for the icon. I am not sure it it's the consequence of the new styles. can you check that?

Reply:
this is fixed in the dashbord ui task